server/knexfile.js

```diff
@@ -0,0 +1,48 @@
+// Update with your config settings.
+
+module.exports = {
+
+  development: {
+    client: 'postgresql',
+    connection: {
+      host: 'localhost',
```

Comment:
Recommend putting these settings in `.env` file and using `process.env.PG_HOST` for example. That way you're not committing sensitive information in your git history.